src/v/raft/types.h

```diff
@@ -349,6 +349,10 @@ struct append_entries_reply
     // older nodes are always ready for recovery.
     bool may_recover = true;
 
+    // hint from follower indicating that it was requested to truncate a log
+    // that was already committed or made visible.
+    bool force_stepdown = false;
```

Comment:
force_stepdown is not set to true anywhere from what I can tell, am I missing something?

Reply:
yes, i missed it during rebase